### PR TITLE
Add IU LaTeX Container Templates

### PR DESCRIPTION
**Pull request by TorbenWetter**

### CODE REVIEW

File: _data/collection-index.yml [hunk 1]
```diff
@@ -258,3 +258,8 @@
   contact: https://github.com/mickeahlinder/devcontainer-features/issues
   repository: https://github.com/mickeahlinder/devcontainer-features
   ociReference: ghcr.io/mickeahlinder/devcontainer-features
+- name: IU LaTeX Container Templates
+  maintainer: Torben Wetter
+  contact: https://github.com/TorbenWetter/iu-latex-container-templates/issues
+  repository: https://github.com/TorbenWetter/iu-latex-container-templates
+  ociReference: ghcr.io/TorbenWetter/iu-latex-container-templates
```
Comment (samruddhikhandale) [inline _data/collection-index.yml]:
```suggestion
  ociReference: ghcr.io/torbenwetter/iu-latex-container-templates
```

The ORAS spec requires the ociReference to be in lower case. Can we update here?
The supporting tools does help convert the cases, however, its better to sync as much as possible.

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]
Thanks, left one comment.

Also, can you help fix the merge conflict?

File: _data/collection-index.yml (hunk 1 above)
Comment (TorbenWetter) [reply _data/collection-index.yml]: Thank you for the feedback!
I have fixed the conflict and updated the `ociReference`.

**Review by samruddhikhandale — APPROVED** [review]
Awesome, thanks! ✨

**Pull request merged**